geth/jail/jail_test.go

```diff
@@ -115,6 +128,46 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+
+	defer func() {
+		if err := recover(); err != nil {
+			require.NotNil(err, "Panic occured")
+		}
+	}()
+
+	require.NotNil(s.jail)
+
+	s.StartTestNode(params.RopstenNetworkID)
+	defer s.StopTestNode()
+
+	// load Status JS and add test command to it
+	s.jail.BaseJS(baseStatusJSCode)
```

Comment:
Not really, other test areas evaluate scripts on to the same jail instance. This may cause unwanted behaviours. See https://github.com/status-im/status-go/pull/194/files/6c9fecb165e3e2ec0331cf37bf418f1fc88fe3ca..ee5cfa909a3faa0e0bf5b00240ff3bbdc845ac52#diff-b79f53878a8f8e28065ebf8f1e6e3f9cR78 and https://github.com/status-im/status-go/pull/194/files/6c9fecb165e3e2ec0331cf37bf418f1fc88fe3ca..ee5cfa909a3faa0e0bf5b00240ff3bbdc845ac52#diff-b79f53878a8f8e28065ebf8f1e6e3f9cR114.